src/Klaviyo.php

```diff
@@ -61,7 +61,13 @@ protected function build_params($params) {
 
     protected function make_request($path, $params) {
         $url = $this->host . $path . '?' . $params;
-        $response = file_get_contents($url);
+        $curl = curl_init();
+        curl_setopt_array($curl, array(
+            CURLOPT_RETURNTRANSFER => 1,
+            CURLOPT_URL => $url,
+            CURLOPT_TIMEOUT => 5 // seconds
```

Comment:
Seems a little high? But I can live with it especially since it's out of our VPC. Just know if we go under 1000ms via `CURLOPT_TIMEOUT_MS `, we need to set an additional option: `CURLOPT_NOSIGNAL => true`.